task/cerberus-check/0.1/test/run.yaml

```diff
@@ -0,0 +1,11 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: TaskRun
+metadata:
+  name: cerberus-check-test
+spec:
+  taskRef:
+    name: cerberus-check
+  params:
+  - name: cerberus_server
+    value: "http://cerberus.chaos-cerberus.svc.cluster.local:8080"
```

Comment:
I wonder if the tests are going to pass as the cerberus server might not be present on the cluster. Probably you'll have to do something of like https://github.com/tektoncd/catalog/blob/main/task/jenkins/0.1/tests/pre-apply-task-hook.sh in order to install the cerberus server for tests

Reply:
Installing cerberus won't be enought, we also need to install an external OCP or K8S cluster in order to configure cerberus to watch it. Do you think it's possible to add this kind of deployment into the pre-apply-task-hook.sh ?
We can imagine to watch the curent cluster in order to avoid deploying another cluster.

Reply:
@vinamra28, does helm command is present in the test environment ?

Reply:
Is it possible to create a namespace into the tekton test environment ? 
Do we have a helm chart CLI into this test environment ? If you have some doc on the test environment in order to develop localy and emulate this testing env (env var availables, directories)

Reply:
> Is it possible to create a namespace into the tekton test environment ? 

@startxfr, yes it's possible to create a namespace in the current cluster but do make sure that it doesn't collides with other namespaces that may get created in future specially when the test suite is running.

> Do we have a helm chart CLI into this test environment ? If you have some doc on the test environment in order to develop localy and emulate this testing env (env var availables, directories)

Probably you can go through the [technical requirements](https://github.com/tektoncd/catalog/blob/main/CONTRIBUTING.md#technical-requirements) which contains some notes around e2e testing. Hope that helps.